docs/data/tree-view/overview/CustomMaterialIcons.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import CloseIcon from '@mui/icons-material/Close';
+import { styled } from '@mui/material/styles';
+import { TreeView } from '@mui/x-tree-view/TreeView';
+import { TreeItem, treeItemClasses } from '@mui/x-tree-view/TreeItem';
+
+const CustomTreeItem = styled(TreeItem)(({ theme }) => ({
+  [`& .${treeItemClasses.content}`]: {
+    padding: theme.spacing(0.5, 1),
+    margin: theme.spacing(0.2, 0),
+    borderRadius: theme.spacing(0.5),
```

Comment:
Feeling like it's a bit weird to use the `spacing` theme token for the border-radius value... wasn't there a token just for that? 🤔

Reply:
I went for this because I wanted a different, softer border radius than the default one defined in the theme (which is 4 I think). In the end it spits out a value in px, but I can change this if it feels weird. We do something similar for the Gmail clone